scout/server/blueprints/institutes/views.py

```diff
@@ -246,12 +246,14 @@ def institute_settings(institute_id):
             return redirect(request.referrer)
 
     data = controllers.institute(store, institute_id)
+    loqus_instances = loqusdb.loqus_ids if hasattr(loqusdb, "loqus_ids") else []
```

Comment:
Populate loqus_instances list after the loqus extension is called and instances are saved